README.md

```diff
@@ -127,6 +157,29 @@ Some providers make available a separately hosted service for handling requests
 
 Used, along with `IdentityHost` to construct a URL that the user is redirected to when initiating the authorization of the service via the backoffice. For GitHub, the required value is `/login/oauth/authorize`.
 
+###### CanManuallyProvideToken
+
+Specifies whether an administrator can manually add tokens.
+
+###### CanManuallyProvideApiKey
+
+Specifies whether an administrator can manually add API keys.
```

Comment:
```suggestion
Specifies whether an administrator can manually add API keys via the backoffice. You might prefer to use this option instead of storing the key in configuration via the `ApiKey` setting.
```